### DISCUSSION TITLE
Mime Type Conflict

### DISCUSSION THREAD

**Discussion by neilgee** [laravel/valet#1131] [Q&A]
I have an intermittent issue in local WordPress development whereby a mime type for CSS is not working and the CSS style borks, the server thinks it is 'text/html' instead of 'text/css' the layout is not correctly rendered, a simple refresh then fixes it - was wondering if there was anything I can try to prevent it happening - example https://share.getcloudapp.com/Z4uYezO0

**Comment by drbyte** [comment]
<img width="828" alt="Screen Shot 2020-06-05 at 5 43 09 PM" src="https://user-images.githubusercontent.com/404472/83925026-1b940b80-a754-11ea-9ea0-d080e682c89f.png">
Why doesn't the .css file have a .css extension? In fact, why doesn't it have a filename at all?

**Comment by neilgee** [comment]
It does , the actual full paths of the 2 files that end in those strings are...
```
/wp-content/uploads/bb-plugin/cache/4f731c9bf83fc03994cbb32c0aa1699b-layout-bundle.js?ver=2.3.2.5-1.3.1
/wp-content/uploads/bb-plugin/cache/89a9d88de2defe9a2dc47c44e58ee036-layout-bundle.css?ver=2.3.2.5-1.3.1
```
So randomly it truncates the URL by missing the full path and loads
```
https://tutorgeo.test/?ver=2.3.2.5-1.3.1
```
Spoke to a BB dev who said its probably a caching issue

**Comment by neilgee** [comment]
The issue below may help me...

> So, if you wish to customize it, note a simple trick with Valet:
All the ~/.config/valet/Nginx/*.* files get loaded inside the nginx http config section, which means you could drop a cache-dev.conf file in there with any directives you want for disabling caching.

https://github.com/laravel/valet/issues/595

Looks like this may work for me, the files I want to exclude from caching appear at the same relative url on various sites `~/Sites/<sitename>/wp-content/uploads/bb-plugin/cache/*.css`

How could I exclude that in a `cache-dev.conf` file
